Add new manual for ServiceChain (#398)
klaytn/klaytn-docs: docs/node/service-chain/getting-started/README.md

---
description: >-

>   Following this chapter, you will quickly set up and run a ServiceChain, an independent blockchain connected to the Klaytn testnet.

Reviewer: ```suggestion
By following the steps in this section, you will quickly set up and run a ServiceChain, an independent blockchain connected to the Klaytn testnet.
```

>   Following this chapter, you will quickly set up and run a ServiceChain, an independent blockchain connected to the Klaytn main chain.

Reviewer 2: ```suggestion
  Following this chapter, you will quickly set up and run a ServiceChain, an independent blockchain connected to the Klaytn testnet.
```

>   By following the steps in this section, you will quickly set up and run a ServiceChain, an independent blockchain connected to the Klaytn testnet.

Reviewer 2: ```suggestion
  By following the steps in this section, you will quickly set up and run a ServiceChain network, an independent blockchain network connected to the Klaytn testnet.
```

Author: Thank you. Git pushed...

---

> This tutorial provides a step-by-step guide for setting up a ServiceChain and connecting it with Klaytn Baobab network.

Reviewer: ```suggestion
This tutorial provides a step-by-step guide for setting up a ServiceChain and connecting it to the Klaytn Baobab network.
```

> This tutorial provides a step-by-step guide for setting up a ServiceChain and connecting it to the Klaytn Baobab network.

Reviewer 2: ```suggestion
This tutorial provides a step-by-step guide for setting up a ServiceChain network and connecting it to the Klaytn Baobab network.
```

Author: Thank you. Git pushed...

Reviewer 2: @nohkwak Seems not updated...

You will also learn how to enable periodic anchoring and cross-chain value transfer.

> In order to construct a blockchain for actual services, we will explain High Availability for ServiceChain and building a ServiceChain hierarchically.

Reviewer: ```suggestion
Regarding blockchains for actual services, we will explain High Availability for ServiceChain and building a ServiceChain hierarchically.
```
